README.md

```diff
@@ -52,10 +52,6 @@ zypper install prometheus-sap_host_exporter
 
 T.B.D.
 
-## Development
-
-T.B.D.
-
```

Comment:
Rather than completely removing this section, what about renaming it to "Contributing" and leaving the T.B.D.?
If you look at how it's done in the sibling project, this section is more intended as an initial contributor guideline (which we could probably copy verbatim from there, because the workflow is exactly the same), but the `devel.md` doc you're sending is more of a collection of development notes, so I would keep the two things separate

Reply:
ok for contributing